libs/core/src/ports/interfaces.ts

```diff
@@ -268,4 +268,9 @@ export interface NotificationsProvider extends Disposable {
   getMessages(
     options: NotificationsProviderGetMessagesOptions,
   ): Promise<NotificationsProviderGetMessagesReturn>;
+
+  registerClientRegistrationToken(
+    userId: number,
+    token: string,
+  ): Promise<unknown>;
```

Comment:
Why is return type `unknown`? Can we generalize the interface to return a boolean or something of the sort?

Reply:
It returns if it is successful otherwise throws an exception... but I can just have it return a boolean.  Doesn't matter though...